SUPPORTED_LANGUAGES.md

```diff
@@ -136,7 +136,7 @@ Languages that listed a **Package** below are 3rd party languages and are not bu
 | Oracle Rules Language   | ruleslanguage          |         |
 | Oxygene                 | oxygene                |         |
 | PF                      | pf, pf.conf            |         |
-| PHP                     | php, php3, php4, php5, php6, php7 |    |
+| PHP                     | php, php3, php4, php5, php6, php7, php8 |    |
```

Comment:
Curious: Does PHP use `php8` or `php5` as file extensions?   I'm not sure why we need adding version numbers here... (I'm not questing THIS particular add, just the pattern)  

We don't do anything different for different versions of PHP. I'm thinking with version 11 all these need to disappear and just have the single "php"? Do you know anything that would suggest otherwise?

Reply:
I'd be hugely in favor to get rid of all version-specific extensions.

As far as I remember, the `php5` extension/mime-type was used in cPanel default configuration options. I don't think anybody continues this pattern anymore. There is no PHP 6 at all either. I think this list was just kept updated by incrementing just this number, but not dropping anything in the name of backwards compatibility?

Reply:
Before my time.  This is a potential breaking change though, so we're stuck with it for now.  But I will add it to the list for v11 changes.